runtime/themes/acme.toml

```diff
@@ -20,6 +20,9 @@
 "diagnostic.error" = {bg="white", modifiers=["bold"]}
 "diagnostic.warning" = {bg="white", modifiers=["bold"]}
 "diagnostic.hint" = {bg="white", modifiers=["bold"]}
+"ui.bufferline" = { fg = "indent", bg = "acme_bar_bg" }
+"ui.bufferline.active" = { fg = "black", bg = "acme_bg" }
+"ui.bufferline.background" = { fg = "indent", bg = "acme_bar_bg" }
```

Comment:
Themes have a fallback behavior: if the theme doesn't define `ui.bufferline.background`, `ui.bufferline` is used instead. So all of these that use the same colors for `ui.bufferline` and `ui.bufferline.background` can be simplified by removing the `ui.bufferline.background` line